recipes/zmsort/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: zmsort
+  version: "1.0"
+
+source:
+  fn: zmsort.tar.gz
+  url: ftp://occams.dfci.harvard.edu/pub/bio/tgi/software/tgicl/zmsort.tar.gz
+  md5: 33e54b4f9b7956c307a2529576f51b10
+
+   
+requirements:
+  build:
+    - gcc   # [not osx]
+    - gclib 
+    - llvm # [osx]
+  run:
+    - libgcc    # [not osx]
+
+test:
+  commands:
+    - zmsort README
```

Comment:
README does not exists in the test/runtime environment. Can we do the same `grep` trick as with the other recipe?